README.md

```diff
@@ -1,5 +1,5 @@
 # AccessLocator
 
-The web application source files are in app/.  This is an application using the Laravel PHP framework.  See its readme for details.
+The web application source files are in app.  This is an application using the Laravel PHP framework.  See its readme for details.
 
-Some tools for gathering location information from other websites have been added to importers/.  These include downloading and CSV generator tools for OpenStreetMap, toiletmap.org and tools for converting CSV into appropriate changes to our seed data files in the Laravel application.
+Some tools for gathering location information from other websites have been added to importers. These include downloading and CSV generator tools for OpenStreetMap, toiletmap.org and tools for converting CSV into appropriate changes to our seed data files in the Laravel application.
```

Comment:
Remove these changes as they aren't related to the issue you're working on.

I suggest you remove the commit by doing:
git rebase -i master
When you see a list of commits, delete the line that says "test for pull requst"
Save and continue.